packages/scandipwa/src/component/Fallback/Fallback.style.scss

```diff
@@ -84,9 +84,11 @@
     }
 
     &-CategoryDetails {
-        display: grid;
-        grid-template-columns: auto minmax(100px, 160px);
-        grid-gap: 12px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        font-size: 14px;
+        font-weight: 400;
```

Comment:
This will make text to CLS . grid is same as category style, it was made to not create less shifts. Especially when image appears.